django/contrib/sessions/backends/base.py

```diff
@@ -148,10 +148,9 @@ def is_empty(self):
 
     def _get_new_session_key(self):
         "Return session key that isn't being used."
-        while True:
+        session_key = get_random_string(32, VALID_KEY_CHARS)
```

Comment:
I'm not sure if refactors like this are useful considering that it makes the code less DRY (`get_random_string(32, VALID_KEY_CHARS)` is repeated). I'd be inclined to leave it as is.

Reply:
Couple of them below looks similar too. I am leaving  them too. Thanks.